pkg/cqrs/sqlitecqrs/sqlitecqrs.go

```diff
@@ -23,14 +28,24 @@ var (
 type SqliteCQRSOptions struct {
```

Comment:
If we allow this to connect to postgres, should we rename the type to something more generic?